### PR TITLE
Feature/list header 5 0

### PR DESCRIPTION
**Pull request by SilverGull**
Brand new header for list. All existing header code replaced.

### CODE REVIEW

File: src/kirby/components/list/list-cell/list-cell.component.ts [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import { Component, OnInit, Input, HostBinding } from '@angular/core';
 
+// TODO: refactor - use map instead of enum
```
Comment (askarby) [inline src/kirby/components/list/list-cell/list-cell.component.ts]: Consider refactoring this yourself. Chances of somebody refactoring this code (later down the road) is "slim to none"

File: src/kirby/components/list/list-cell/list-cell.component.ts [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 enum verticalAlignmentEnum {
   top = 'flex-start',
   center = 'center',
   bottom = 'flex-end',
   stretch = 'stretch',
   baseline = 'baseline',
 }
+// TODO: refactor - use map instead of enum
```
Comment (askarby) [inline src/kirby/components/list/list-cell/list-cell.component.ts]: See comment above!

File: src/kirby/components/list/list.component.tns.html [hunk 3]
```diff
@@ -1,15 +1,8 @@
-<GridLayout [rows]="rowDefinition(headerTemplate)">
-  <ng-container *ngIf="headerTemplate">
-    <FlexboxLayout alignItems="center" class="list-header-container" row="0">
-      <ng-container
-        *ngTemplateOutlet="headerTemplate; context: { $implicit: columns }"
-      ></ng-container>
-    </FlexboxLayout>
-  </ng-container>
+<GridLayout rows="*"> <!-- GridLayout kept for backwards compatability-->
```
Comment (askarby) [inline src/kirby/components/list/list.component.tns.html]: Typo (really a non-issue), it's written: "compatibility"

File: src/kirby/components/list/list-cell/list-cell.component.ts (hunk 1 above)
Comment (SilverGull) [reply src/kirby/components/list/list-cell/list-cell.component.ts]: agree - will make a new branch

File: src/kirby/components/list/list-cell/list-cell.component.ts (hunk 2 above)
Comment (SilverGull) [reply src/kirby/components/list/list-cell/list-cell.component.ts]: agree - will make a new branch

File: src/kirby/components/list/list-cell/list-cell.component.ts (hunk 1 above)
Comment (SilverGull) [reply src/kirby/components/list/list-cell/list-cell.component.ts]: agree - will make a new branch

File: src/kirby/components/list/list-cell/list-cell.component.ts (hunk 2 above)
Comment (SilverGull) [reply src/kirby/components/list/list-cell/list-cell.component.ts]: agree - will make a new branch

**Pull request merged**